cmd/create/operatorroles/cmd.go

```diff
@@ -542,3 +561,22 @@ func validateOperatorRoles(r *rosa.Runtime, cluster *cmv1.Cluster) ([]string, er
 	}
 	return missingRoles, nil
 }
+
+func validateOperatorRolesMatchOidcProvider(r *rosa.Runtime, cluster *cmv1.Cluster) error {
```

Comment:
This function is very similar to `validateOperatorRolesMatchOidcProvider` in `cmd/create/cluster/cmd.go`.
Can we extract the function to a common file and reuse it?

Reply:
Sort of, had to adapt it as the operator roles types are different. But it does work